file-ingestion-test/src/main/java/com/hazelcast/jet/tests/file/ingestion/FileIngestionTest.java

```diff
@@ -150,16 +164,12 @@ private Pipeline pipeline(JobType jobType, int jobNumber) {
             case HDFS:
                 return FileSources.files(hdfsPath)
                         .option("fs.defaultFS", hdfsUri)
```

Comment:
The property `fs.defaultFS` should not be needed either, if you use full url in `hdfsPath`